lib/preflight/preflighttest.ts

```diff
@@ -438,6 +456,11 @@ export class PreflightTest extends EventEmitter {
     }
     return collectedStats;
   }
+
+  private _updateProgress(name: string): void {
+    this._progressEvents.push({ timestamp: Date.now(), name });
```

Comment:
instead of  `timestamp` we should use timeFromStart as it would be much useful that absolute time.